src/node_file.cc

```diff
@@ -2705,6 +2705,29 @@ static void Chmod(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void ChmodSync(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  const int argc = args.Length();
+  CHECK_GE(argc, 2);
+
+  BufferValue path(env->isolate(), args[0]);
+  CHECK_NOT_NULL(*path);
+  THROW_IF_INSUFFICIENT_PERMISSIONS(
+      env, permission::PermissionScope::kFileSystemWrite, path.ToStringView());
+
+  CHECK(args[1]->IsInt32());
+  int mode = args[1].As<Int32>()->Value();
+
+  uv_fs_t req;
+  auto make = OnScopeLeave([&req]() { uv_fs_req_cleanup(&req); });
```

Comment:
```suggestion
  auto cleanup = OnScopeLeave([&req]() { uv_fs_req_cleanup(&req); });
```
nit: a more meaningful name